src/parsers/helper.ts

```diff
@@ -23,6 +23,18 @@ export const supportedFileType = (filePath: string): 'ts' | 'js' | undefined =>
   return undefined;
 };
 
+/**
+ * determine if the file is a react file(jsx, tsx);
+ * @param filepath
+ * @returns true or false
+ */
+export const isReactFileType = (filePath: string): boolean => {
+  if (filePath.match(/\.[jt]sx$/)) {
+    return true;
+  }
+  return false;
```

Comment:
The conditional return is the same as returning the condition. Also, you likely want case insensitivity with `/i`.
```suggestion
  return filePath.match(/\.[jt]sx$/i);
```

Reply:
updated logic to check only against ts.